pkg/cvo/availableupdates.go

```diff
@@ -432,36 +429,71 @@ func unknownExposureMessage(risk configv1.ConditionalUpdateRisk, err error) stri
 	return fmt.Sprintf(template, risk.Name, err, risk.Name, risk.Message, risk.Name, risk.URL)
 }
 
-func evaluateConditionalUpdate(ctx context.Context, conditionalUpdate *configv1.ConditionalUpdate, conditionRegistry clusterconditions.ConditionRegistry) *metav1.Condition {
-	recommended := &metav1.Condition{
-		Type: ConditionalUpdateConditionTypeRecommended,
+func newRecommendedStatus(now, want metav1.ConditionStatus) metav1.ConditionStatus {
+	switch {
+	case now == metav1.ConditionFalse || want == metav1.ConditionFalse:
+		return metav1.ConditionFalse
+	case now == metav1.ConditionUnknown || want == metav1.ConditionUnknown:
+		return metav1.ConditionUnknown
+	default:
+		return want
 	}
-	messages := []string{}
-	for _, risk := range conditionalUpdate.Risks {
+}
+
+const (
+	recommendedReasonRisksDoNotApply  = "RisksDoNotApply"
+	recommendedReasonEvaluationFailed = "EvaluationFailed"
+	recommendedReasonMultiple         = "MultipleReasons"
+
+	// recommendedReasonRiskApplies is used instead of the original name if it
+	// does not match the pattern for a valid k8s condition reason.
+	recommendedReasonRisksApply = "RisksApplyToCluster"
```

Comment:
Nitpick: The reason `RisksApplyToCluster` sounds fine , but it is not consistent with the other strings i.e. `RisksDoNotApply` , `MultipleReasons` , `MultipleReasons` . Do `ReleventRisks` or `ApplicableRisk` sounds consistent or better?

Reply:
IMO the wording is important because it is customer facing as console or OC CLI might show these reasons directly to users.

Reply:
I am happy to change `RisksDoNotApply` to `RisksDoNotApplyToCluster` for consistency. 

I don't think there needs to be any special consistency with other reasons, I think both `EvaluationFailed` and `MultipleReasons` work well in the contexts when we set them?

Reply:
What about `ExposedToRisks` and `NotExposedToRisks`?

Reply:
ExposedRisks and NoExposedRisks sounds good. If not then ExposedToRisks and NotExposedToRisks would ok too.

Reply:
I did the `ExposedToRisks` pair